spec/classes/pulp_apache_spec.rb

```diff
@@ -172,8 +172,28 @@
     Options FollowSymLinks Indexes
 </Directory>
 
-# -- HTTPS ISOS
-Alias /pulp/isos /var/www/pub/https/isos
+# -- GPG Keys -------------------
+Alias /pulp/gpg /var/www/pub/gpg
+
+<Directory /var/www/pub/gpg/>
+    Options FollowSymLinks Indexes
+</Directory>
+')
+
+        is_expected.to contain_file('/etc/pulp/vhosts80/rpm.conf').with(
```

Comment:
```verify_exact_contents(catalogue, '/etc/pulp/vhosts80/rpm.conf', ['Alias /pulp/exports /var/www/pub/yum/http/exports'])``` should be equal and easier to read. There are more cases in this file that could benefit from it.